js/app.js

```diff
@@ -16,4 +18,108 @@ navLinks.forEach((link) => {
 
     navMenu.classList.remove('active');
   });
-});
\ No newline at end of file
+});
+
+const projects = [
+  {
+    id: 1,
+    name: 'Tonic',
+    description:
+      'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry standard  ',
+    image: '../images/work-image1.png',
+    technologies: ['html', 'css', 'JavaScript'],
+    link: 'https://github.com/Ukaypromise/Personal-Portfolio',
+    source: 'https://github.com/Ukaypromise/Personal-Portfolio',
+  },
+  {
+    id: 2,
+    name: 'Multi-Post Stories',
+    description:
+      'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the1500, when an unknown printer took a galley of type and scrambled it 1960s.',
+    image: '../images/work-image2.png',
+    technologies: ['html', 'css', 'JavaScript'],
+    link: 'https://github.com/Ukaypromise/Personal-Portfolio',
+    source: 'https://github.com/Ukaypromise/Personal-Portfolio',
+  },
+  {
+    id: 3,
+    name: 'Facebook 360',
+    description:
+      ' Lorem Ipsum han printer took a galley of type and scrambled it 1960s with the releawn printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. scrambled it 1960s with the relea',
+    image: '../images/work-image3.png',
+    technologies: ['html', 'css', 'JavaScript'],
+    link: 'https://github.com/Ukaypromise/Personal-Portfolio',
+    source: 'https://github.com/Ukaypromise/Personal-Portfolio',
+  },
+  {
+    id: 4,
+    name: 'Uber Navigation',
+    description:
+      'Lorem Ipsum has been the industry standard dummy text ever since the 1500s, when an unknown printer took a galley.',
+    image: '../images/work-img4.png',
+    technologies: ['html', 'css', 'JavaScript'],
+    link: 'https://github.com/Ukaypromise/Personal-Portfolio',
+    source: 'https://github.com/Ukaypromise/Personal-Portfolio',
+  },
+];
+
+const onModalOpen = async (id) => {
+  const getData = await projects.find((project) => project.id === id);
+
+  const modal = document.querySelector('.modal-container');
+  modal.style.display = 'block';
+  document.getElementById('modal-title').innerHTML = getData.name;
+  document.getElementById('modal-desc').innerHTML = getData.description;
+  document.getElementById('modal-img').src = getData.image;
+  document.getElementById('modal-see-live').src = getData.link;
+  document.getElementById('modal-see-source').src = getData.source;
+
+  document.getElementById('group-list').innerHTML = getData.technologies
+    .map((tech) => `<li class="stack">${tech}</li>`)
+    .join('');
+};
+const p = onModalOpen;
+
+const onModalClose = () => {
+  const modal = document.querySelector('.modal-container');
+  modal.style.display = 'none';
+};
+onModalClose(p);
+
+window.addEventListener('DOMContentLoaded', () => {
```

Comment:
- [ ] As the previous reviewer mentioned, your popup needs to appear **only when the user clicks on the see project button in my work section.** Your popup is still being shown directly when the browser loads the page. I bet this is because **you have used an inappropriate event which is DOMContentLoaded.** This event causes the function to be called every time the browser finishes loading HTML. **I think you can put this event listener to the see project button instead, and then pass in the click event. 😃 👍 OR you can remove it. Just make sure that the popup is not displayed directly after the page is loaded**